### DISCUSSION TITLE
Do you have to put ExtPay.js in the background.js?

### DISCUSSION THREAD

**Discussion by synapticloop** [Glench/ExtPay#157] [General]
It says in the documentation:

`Add ExtPay to background.js (required!)`

I hadn't implemented it and was testing (load unpacked extension) without it and it seems to work fine - what doesn't work if it isn't included?

**Comment by Glench** [comment]
You can check the source code to see, but the onpaid listeners especially wouldn't work, and potentially any use of ExtPay in content scripts.

**Reply by synapticloop** [reply]
Fair enough - I don't do either - thanks!
